common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5734,6 +5734,26 @@ public static enum ConfVars {
             + "a background update happens periodically to report the actual state of the query"),
     HIVE_SCHEDULED_QUERIES_CREATE_AS_ENABLED("hive.scheduled.queries.create.as.enabled", true,
         "This option sets the default behaviour of newly created scheduled queries."),
+    HIVE_QUERY_HISTORY_SERVICE_ENABLED("hive.query.history.service.enabled", true,
```

Comment:
could we remove `service` from the vars and configs, I think it's redundant?

Reply:
hm, makes sense